src/Downloads.jl

```diff
@@ -395,4 +399,13 @@ arg_read_size(io::Base.GenericIOBuffer) = bytesavailable(io)
 arg_read_size(::Base.DevNull) = 0
 arg_read_size(::Any) = nothing
 
+function content_length(headers)
```

Comment:
```suggestion
function content_length(headers::Union{AbstractVector, AbstractDict})
```
I like to clarify the type even on helper definitions like this (mostly just for documentation).